packages/nextjs/services/api/sre/schema.ts

```diff
@@ -0,0 +1,15 @@
+export type SocialLinks = {
+  socialX?: string;
+  socialGithub?: string;
+  socialDiscord?: string;
+  socialTelegram?: string;
+  socialInstagram?: string;
+  socialEmail?: string;
+};
+
+export type BuilderData = {
+  userAddress: string;
+  role?: "anonymous" | "user" | "admin";
+  socialLinks?: SocialLinks;
+  batchId?: { number: string; status: string };
```

Comment:
For the future. We could have the proper type for the return from SRE

```
  userAddress: '',
  role: '',
  createdAt: '',
  updatedAt: ',
  ens: '',
  ensAvatar: '',
  socialTelegram: '',
  socialX: '',
  socialGithub: '',
  socialInstagram: '',
  socialDiscord: '',
  socialEmail: '',
  socialFarcaster: ',
  location: '',
  batchId: '',
  batchStatus: '',
  referrer: '',
  originalUtmParams: ''
```

and use it in the app (instead of remapping some stuff in `fetchBuilderData`)